conf/math_code.yaml

```diff
@@ -0,0 +1,58 @@
+defaults:
+  - base
+  - /domain_rollouts@domain_rollouts: base
+  - domain_mix: math_coding_70_30
+  - _self_
+
+actor:
+  rollout_policy: pipelinerl.domains.dispatcher.generate_multidomain_rollout
+  system_prompt: ""
+  task_template: |-
+    {task}
+  task_prompt: ""
+  ensure_boxed_answers: false
+  domain_rollouts:
+    math: ${domain_rollouts.math}
+    coding: ${domain_rollouts.coding}
+  coding_time_limit_s: 15.0
+  coding_per_test_timeout_s: 10.0
+  coding_memory_limit_bytes: 1073741824
+  coding_compile_timeout_s: 10.0
+  coding_sandbox_url: ${oc.env:CODING_SANDBOX_URL, "http://sandbox:8080/run_code"}
+
+dataset_loader: pipelinerl.domains.multidomain.loader.load_datasets
```

Comment:
Can we have multiple dataloaders at once so we load different datasets for different domains in the same exp?